harness/determined/common/experimental/experiment.py

```diff
@@ -289,3 +303,11 @@ def key(ckpt: checkpoint.Checkpoint) -> Any:
 
     def __repr__(self) -> str:
         return "Experiment(id={})".format(self.id)
+
+    @classmethod
+    def _from_bindings(
```

Comment:
This is probably a stupid style thing, but it makes me happy to see constructors grouped together. Do you have an active preference on keeping this down at the bottom of the class definition (instead of following `__init__`, or did it just happen that way?

Reply:
I don't have a strong preference, but my rough reasoning was that methods like `__repr__` are at the bottom, and those are also "class-level" methods. i guess i thought of it more like a class-level method and not a constructor.